File: 5 - filas/python/cliente.py

class Cliente:  
    def __init__(self, fone, nome, ficha):
        self.fone = fone
        self.nome = nome
        self.ficha_atendimento = ficha
    
    def __str__(self):
        return "Cliente: " + self.nome + " : " + self.fone + " -> Ficha: " + self.ficha_atendimento + "\n"

File: 05 - filas/java/Cliente.java

public class Cliente {
    public String fone; //55996619445    
    public String nome;
    public int fichaAtendimento;

    public Cliente(String fone, String nome, int ficha){
        this.fone = fone;
        this.nome = nome;
        this.fichaAtendimento = ficha;
    }

    @Override
    public String toString() {
        return "Cliente: " + this.nome + " : " + this.fone + " -> Ficha: " + this.fichaAtendimento + "\n";
    }
}
